src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/changelist/cessations/CorporateManagingOfficerCessation.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.cessations;
+
+import java.time.LocalDate;
+
+public class CorporateManagingOfficerCessation extends ManagingOfficerCessation {
+
+    public CorporateManagingOfficerCessation(String officerAppointmentId,
+                                             LocalDate actionDate,
+                                             String officerName) {
+        super(officerAppointmentId, officerName, actionDate);
+        setAppointmentType("Corporate MO");
```

Comment:
Just going by what we have so far
```suggestion
        setAppointmentType("Managing Officer");
```

Reply:
Just realised you've added this as per AC, let's check with @AndreKramer7654

Reply:
My understanding is that there is only one appointment type for MOs (Individual and Corporate) and it's "Managing Officer".

Reply:
Confirmed - leave them in for now but effectively they are not needed.